### PR TITLE
database to main

### PR DESCRIPTION
**Pull request by hhorace**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,47 +1,93 @@
 require 'rake/testtask'
 
 task :default do
-    puts 'rake -T'
+  puts 'rake -T'
```
Comment (chinyakao) [inline Rakefile:4]: It should be `` puts `rake -T` ``.
you can just copy and paste see the different.
(Notice that the quotation marks should be the one next to the keyboard number 1 not the one next to the keyboard enter.)

File: app/infrastructure/database/orms/groups.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      #             class: :'NoFB::Database::PostsOrm',
+      #             join_table: :projects_members,
+      #            left_key: :member_id, right_key: :project_id
+
+      plugin :timestamps, update_on_create: true
+
+      def self.find_or_create(member_info)
+        GroupsOrm.strict_param_setting = false
+        first(group_id: member_info[:group_id]) || insert(member_info)
+        # puts GroupsOrm[group_id: member_info[:group_id]]
```
Comment (chinyakao) [inline app/infrastructure/database/orms/groups.rb:24]: It looks like the code is used to debug. please try not to upload to version control

File: app/infrastructure/database/orms/posts.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+      plugin :timestamps, update_on_create: true
+
+      def self.find_or_create(member_info)
+        PostsOrm.strict_param_setting = false
+        # puts member_info
+        # puts member_info[:post_id]
+        # puts PostsOrm.dataset.all
+        # puts first(post_id: member_info[:post_id])
+        # puts create(member_info)
```
Comment (chinyakao) [inline app/infrastructure/database/orms/posts.rb:25]: It looks like the code is used to debug. please try not to upload to version control

File: app/infrastructure/database/orms/users.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+      plugin :timestamps, update_on_create: true
+
+      def self.find_or_create(member_info)
+        UsersOrm.strict_param_setting = false
+        # puts UsersOrm.columns
+        # puts member_info
+        # puts member_info[:user_id]
+        # puts first(user_id: member_info[:user_id])
+        # puts insert(member_info)
```
Comment (chinyakao) [inline app/infrastructure/database/orms/users.rb:27]: It looks like the code is used to debug. please try not to upload to version control

**Review by chinyakao — COMMENTED** [review]
Please take a look
@hhorace @ginagigo123 @lauralishishihoohoo

**Comment by chinyakao** [comment]
`rake spec` also has some errors
@hhorace @ginagigo123 @lauralishishihoohoo

**Comment by ginagigo123** [comment]
> `rake spec` also has some errors @hhorace @ginagigo123 @lauralishishihoohoo

We have already fixed the errors in #9 .

**Comment by chinyakao** [comment]
Thanks for replying me

**Pull request merged**